back/app/logging_config.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+
+def setup_logging():
+    logging.basicConfig(
+        level=logging.DEBUG,
+        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+        handlers=[logging.FileHandler("app.log"), logging.StreamHandler()],
```

Comment:
Il faut plutot mettre les logs dans la sortie console

Reply:
Prends exemple sur le projet data etat : 
https://github.com/dataregion/DataEtatRegion-back/blob/f862bfb5e6bfdde89136009b2f26f344976e38f7/app/__init__.py#L65

Reply:
Grace aux handlers, nous pouvons orienter la sortie des logs vers un fichier grâce à logging.FileHandler("app.log") ou vers la console via logging.StreamHandler() : https://www.toptal.com/python/in-depth-python-logging - https://docs.python.org/3/library/logging.handlers.html